lib/runner/concurrency/index.js

```diff
@@ -362,12 +362,7 @@ class Concurrency extends EventEmitter {
     let workers = require('os').cpus().length;
 
     if (isObject(test_workers) && isNumber(test_workers.workers)) {
-      if (workers < test_workers.workers) {
-        // eslint-disable-next-line no-console
-        Logger.warn(`Number of max workers is set to ${test_workers.workers} while the number of cpu cores is ${workers}. This can cause performance issues...\n`);
-      } else {
-        workers = test_workers.workers;
-      }
+      workers = test_workers.workers;
```

Comment:
we can keep the warning if `start_process: true` and maybe add small test as well?

Reply:
I could add a warning but I am not sure about the number of worker threads above they should be warned because it seems it really depends on what kind of tests they have and how will impact the CPU core.

I added a test for the changes.